.github/pr-custom-review.yml

```diff
@@ -17,18 +17,11 @@ rules:
   - name: Audit rules
     check_type: changed_files
     condition:
-      include: ^polkadot/runtime\/(kusama|polkadot|common)\/.*|^polkadot/primitives/src\/.+\.rs$|^substrate/primitives/.*|^substrate/frame/.*
-      exclude: ^polkadot/runtime\/(kusama|polkadot)\/src\/weights\/.+\.rs$|^substrate\/frame\/.+\.md$
-    all_distinct:
-      - min_approvals: 1
-        teams:
-          - locks-review
-      - min_approvals: 1
-        teams:
-          - polkadot-review
-      - min_approvals: 1
-        teams:
-          - srlabs
+      include: ^polkadot/runtime\/(kusama|polkadot|common)\/.*|^polkadot/primitives/src\/.+\.rs$|^substrate/primitives/.*
```

Comment:
This is still wrong... `locks-review` was about protecting 
:lock: :lock: :lock: 
line of code
:lock: :lock: :lock:

Reply:
What team should I set here?

> So, we probably should only keep locks-review and they can operate on the entire code base for when "locked" code is changed.

We have miss interpreted your message, I thought you referred to using the `lock-teams`.

Should I simply remove this rule?

Reply:
well it just moved and merged all together from 3 repos, there's high chance that it could be done better
Lets revisit these pr-cr settings in scope of monorepo holistically 

so in order to implement properly https://github.com/paritytech/pr-custom-review/issues/136 this task, with post-actions like adding a project etc we will need to wait until https://github.com/paritytech/review-bot/milestone/1 is done and we can integrate it here with all new rules

before it's done i think this whole condition from https://github.com/paritytech/pr-custom-review/issues/136 could be removed for now, and keep only the old polkadot one https://github.com/paritytech/polkadot/blob/master/.github/pr-custom-review.yml#L10-L11

Reply:
Just remove the rules here entirely for now.

Reply:
@bkchr you mean all PRCR rules or only Audit?
if all - why?